README.md

```diff
@@ -19,3 +19,9 @@ most important part. Add your answer to this markdown file.
 
 Implement a function to convert an adjacency list to an adjacency matrix and
 analyze it as above.
+
+I used ai to give me a rough psuedo code so I could understand how a matrix and a list might interact.
```

Comment:
How similar was that to what you submitted?

Reply:
I suppose pretty similar but since it was a rough psuedo code it didnt give me too much to go off of just the ideas and i wrote the code.

Reply:
Can you provide the pseudocode and the prompt you used please?